test/caesar_test.rb

```diff
@@ -1,2 +1,32 @@
+require 'minitest/pride'
+require 'minitest/autorun'
+require './lib/caesar.rb'
+
+class CaesarTest < MiniTest::Test
+
+  def test_it_exits
+    caesar = Caesar.new("the quick brown fox jumps over the lazy dog")
+    assert_instance_of Caesar, caesar
+  end
+
+  def test_translate_english_to_ciphertext
+    caesar = Caesar.new("the quick brown fox jumps over the lazy dog")
+    assert_equal "the quick brown fox jumps over the lazy dog", caesar.eng_to_cipher
+  end
+
+  def test_it_convert_string_to_array
+    caesar = Caesar.new("the quick brown fox jumps over the lazy dog")
+    assert_equal [], caesar.split
```

Comment:
This is the right thought process... you have a method split, so you should test it. The assertion is not quite right though. What should the split method return? Probably not an empty array.